toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -231,9 +231,9 @@ if [[ "${version}" == "3.0" ]]; then # Add 3.0 DailyBuild repo
     cp resources/mariner-3_repo $tmp_dir/mariner-3_repo
     sed -i "s~<DAILY_BUILD_ID>~${DAILY_BUILD_ID}~" $tmp_dir/mariner-3_repo
     if [[ $(uname -m) == "x86_64" ]]; then
-        sed -i "s~<ARCH>~amd64~" $tmp_dir/mariner-3_repo
+        sed -i "s~<ARCH>~x86-64~" $tmp_dir/mariner-3_repo
```

Comment:
Instead of hard-coding the architecture string, can you try something like the following to get the URL

```
wget -nv https://mariner3dailydevrepo.blob.core.windows.net/lkg/lkg-3.0-dev.json
cat lkg-3.0-dev.json | jq -r .repo.$(uname -p)
```

Reply:
Just watch out, the entry in there has dash for `x86-64` and `$(uname -p)` returns it as `x86_64`. So you would need to make a variable:
`AZURE_FORMAT_ARCHITECTURE=$(uname -p)`
`AZURE_FORMAT_ARCHITECTURE=${ARCHITECTURE//_/-}`